cypress/e2e/testAddAdminDescription.cy.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types='cypress' />
+
+let people;
```

Comment:
Is it work? I can't find 
cy.fixture('users.json').then(function(data) {
            people = data.users